client/.pylintrc

```diff
@@ -81,7 +81,8 @@ disable=raw-checker-failed,
         suppressed-message,
         useless-suppression,
         deprecated-pragma,
-        use-symbolic-message-instead
+        use-symbolic-message-instead,
+        cyclic-import
```

Comment:
What did we need to add `cyclic-import` here?

Reply:
Without it: lint error with:
```
lint: commands[1]> pylint -rn qiskit_serverless tests
************* Module tests.core.__init__
tests/core/__init__.py:1:0: R0401: Cyclic import (qiskit_serverless.core.function -> qiskit_serverless.core.job) (cyclic-import)
```

Reply:
https://github.com/Qiskit/qiskit-serverless/pull/1480#issuecomment-2321486321

Reply:
One question, Aki. If you do this local import: https://github.com/Qiskit/qiskit-serverless/pull/1480#discussion_r1742578677

does it continue being needed to add the `cyclic-import`?

Reply:
Yes, it does.  Some tests fail like "tests/core/__init__.py:1:0: R0401: Cyclic import".  I could not make ignore it by the comment because it's an empty file.